### PR TITLE
Implemented order receipt/cancellation message

### LINKED ISSUE #39: display a receipt message in the browser window after a successful CyberSource callback
part of #5 As a learner, I want to get a confirmation that the payment was received, after each of my payments.

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #39 

#### What's this PR do?
Pops up a modal when there's a successful payment, when the user cancelled the payment, and when two minutes have passed without determining if the payment went through.

#### How should this be manually tested?
- Go to `/pay/?status=cancel` and you should see the cancellation message
- Make a payment, then get a reference to the `Order` in the shell. Change the status from `fulfilled` to `created` and save.
- Go to `/pay/?status=receipt&order=order_num` replacing `order_num` with `order.id` and look at the console logs. You should see it refetching the klass API every 3 seconds or so. 
- In the shell change the status back to `fulfilled` and resave the order. After 3 seconds or so you should see a success toast message pop up.
- If you waited two minutes without changing the order status you should see a failure message and no more attempts at refetching the klasses API.

### CODE REVIEW

**Comment by giocalitri** [comment]
Code looks fine to me: checking functionality

**Comment by giocalitri** [comment]
nit: The message background color is too close to the page color

File: static/js/containers/PaymentPage.js [hunk 1]
```diff
@@ -95,6 +109,118 @@ class PaymentPage extends React.Component {
...
+      this.handleOrderCancellation();
+    }
+  };
+
+  handleOrderSuccess = (klass: Object): void => {
+    const { dispatch, ui: { toastMessage } } = this.props;
+    if (toastMessage === null) {
+      dispatch(setToastMessage({
+        title: 'Order Complete!',
+        message: `You are now enrolled in ${klass.full_klass_title}`,
```
Comment (giocalitri) [inline static/js/containers/PaymentPage.js]: This message probably does not make much sense. We should say something like "you made a successful payment for Bootcamp FooBar" or something like this.
Maybe ask Rob

**Comment by giocalitri** [comment]
Functionality is fine: just 2 small changes.

**Comment by noisecapella** [comment]
@giocalitri @roberthouse54 thinks the background color is fine, he's looking over the text now

**Comment by giocalitri** [comment]
@noisecapella the background of the message? Is it visible enough?

**Comment by noisecapella** [comment]
Yes, I showed it to him

**Comment by noisecapella** [comment]
@giocalitri I changed the copy to 'Payment Complete!' per @roberthouse54's suggestion. I think that should be everything

**Comment by giocalitri** [comment]
👍 :shipit:

**Pull request merged**